includes/snmp.inc.php

```diff
@@ -127,7 +127,7 @@ function gen_snmpget_cmd($device, $oids, $options = null, $mib = null, $mibdir =
  * @param string $mibdir a mib directory to search for mibs, usually prepended with +
  * @return array the fully assembled command, ready to run
  */
-function gen_snmpwalk_cmd($device, $oids, $options = null, $mib = null, $mibdir = null)
+function gen_snmpwalk_cmd($device, $oids, $options = null, $mib = null, $mibdir = null, $communityStringIndexing = 0)
```

Comment:
Hi @klui2k1 
Instead of zero, you should use null here, as the other optional parameters. Cause 0 might be a valid communityStringIndex, depending on the device.

Reply:
Yeah, that's a good point. I will make the change. I will need to study how I retrieve your commits so my local repo is synced.